### ISSUE TITLE
XPUB                :  [5183a8d8/86'/1'/0']

### ISSUE THREAD

**Issue by wyq000** [bitlightlabs/bitlight-local-env-public#6]
How is 5183a8d8 generated?

**Comment by bitlightlabs-dev** [comment]
```
git clone https://github.com/bitlightlabs/bitlight-local-env-public/
make up
make alice-cli
```

The Fingerprint is 5183a8d8.

```
Network             :  regtest
Wallet Name         :  alice
Root XPRV           :  tprv8ZgxMBicQKsPeHzjP5LTL818LxwHbJNLZRa98Qdnn7M98fW15365cB1Sz9QZvYufASRKH6JEPhfpxVuFTKMHxDcEAVboqKuZdMmxzKVhMnW
Fixed XPUB          :  [5183a8d8/86'/1'/0']tpubDDtdVYn7LWnWNUXADgoLGu48aLH4dZ17hYfRfV9rjB7QQK3BrphnrSV6pGAeyfyiAM7DmXPJgRzGoBdwWvRoFdJoMVpWfmM9FCk8ojVhbMS/<0;1;9;10>/*
Bitcoin Address     :  bcrt1pn0s2pajhsw38fnpgcj79w3kr3c0r89y3xyekjt8qaudje70g4shs20nwfx
```

**Comment by wyq000** [comment]
> ```
> git clone https://github.com/bitlightlabs/bitlight-local-env-public/
> make up
> make alice-cli
> ```
> 
> 指纹是 5183a8d8。
> 
> ```
> Network             :  regtest
> Wallet Name         :  alice
> Root XPRV           :  tprv8ZgxMBicQKsPeHzjP5LTL818LxwHbJNLZRa98Qdnn7M98fW15365cB1Sz9QZvYufASRKH6JEPhfpxVuFTKMHxDcEAVboqKuZdMmxzKVhMnW
> Fixed XPUB          :  [5183a8d8/86'/1'/0']tpubDDtdVYn7LWnWNUXADgoLGu48aLH4dZ17hYfRfV9rjB7QQK3BrphnrSV6pGAeyfyiAM7DmXPJgRzGoBdwWvRoFdJoMVpWfmM9FCk8ojVhbMS/<0;1;9;10>/*
> Bitcoin Address     :  bcrt1pn0s2pajhsw38fnpgcj79w3kr3c0r89y3xyekjt8qaudje70g4shs20nwfx
> ```
I want to know the principle of this generation. What parameters are used to encode the 5183a8d8 fingerprint?
